src/components/FeatureViews/Registration/RegistrationLanding.js

```diff
@@ -1,7 +1,8 @@
 import * as hooks from "actions/hooks";
-import React, {useCallback, useMemo, useState} from "react";
+import { distinctObjectArray, gradeOptions } from "../../../utils";
```

Comment:
You can also use file paths relative to the `src/` directory for imports:

    import {...} from "utils"; // utils is at the top level inside src